### ISSUE TITLE
Youtube videos consistently freeze at 25:21 intervals 

### ISSUE THREAD

**Issue by Feodor2** [Feodor2/Mypal#8]
It occurs on winxp only, this is weird, certainly this was inherited from mozzila.
https://productforums.google.com/forum/?_escaped_fragment_=topic/youtube/ViiQ_4nNXn8#!topic/youtube/ViiQ_4nNXn8
It may be realted to inserting ads during palyback, that obnoxious google adopted on youtube.
Now searching winxp browser which not has that problem.

**Comment by kitaro1** [comment]
I downloaded cometbird browser 11 polish. A lot of pages still work on this dead browser. Even YouTube does not need much http://prntscr.com/im6dan
The problem is with github, but other websites have no problems: http://prntscr.com/im6dkj
Cometbird was extremely light and it would be a good proposition to modify and adapt to today's Internet functions.
?

**Comment by Atavic** [comment]
Yes, I get the same. If it's exactly 25:21 you can remember the timelapse, multiply it and then reload everytime you get to 25:21, 50:42 etc.

**Comment by Feodor2** [comment]
If to reload it will plays from the beginning, i do another way, move slightly seek bar and video continues.

**Comment by JoachimHenze** [comment]
A different workaround could be, say this is your youtube URL e.g:
https://www.youtube.com/watch?v=0sfYKFYqdZw
just add a "minus" after the t
https://www.yout-ube.com/watch?v=0sfYKFYqdZw

This will get you rid of all ads, eventually it helps to workaround the bug.
